uaclient/contract.py

```diff
@@ -295,17 +295,19 @@ def request_updated_contract(
                 allow_enable=allow_enable,
             )
         except exceptions.UserFacingError as e:
-            user_errors.append(e)
+            with util.disable_log_to_console():
+                logging.exception(str(e))
```

Comment:
A nit, but `str(e)` isn't very useful here; we should get the exception message as part of the stack trace regardless. Better would be logging some additional information that _isn't_ in the message.